changelog/next/features/3684--contextualizer-backend.md

```diff
@@ -0,0 +1,8 @@
+The closed-source `contextualizer` plugin offers a backend functionality for
```

Comment:
I think this one's just called `context`, as that's also the operator name?

Reply:
The context plugin is built into a `libtenzir-plugin-contextualizer.so` library, so the user needs to specify the `contextualizer` plugin name.
I didn't change it because `contextualizer` is more meaningful than `context`, IMO.
If we drop all the `context` suffixes from the contexts, we can also drop the `-ualizer` here.

Reply:
I renamed the project in https://github.com/tenzir/tenzir-plugins/pull/134/commits/a29a64f262e1847f1e81b9658477b67cfde326be

Reply:
Thanks! 🙏